Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -213,6 +213,9 @@ MediaPlayerPrivateGStreamer::~MediaPlayerPrivateGStreamer()
         ASSERT(bus);
         gst_bus_set_sync_handler(bus.get(), nullptr, nullptr, nullptr);
         g_signal_handlers_disconnect_matched(m_pipeline.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
+        // BRCM specific, flush to unlock aud/vid filters
```

Comment:
Ok let's remove that patch for now then.
The fix I'm suggesting is to define WTF_PLATFORM_BRCM to 1 during the build, so that you can use #if PLATFORM(BRCM) in the code.